plasmapy/physics/parameters.py

```diff
@@ -394,8 +394,7 @@ def thermal_speed(T, particle="e", method="most_probable"):
     try:
         m = ion_mass(particle)
     except Exception:
-        raise ValueError("Unable to find {} mass in thermal_speed"
-                         .format(particle))
+        raise ValueError("Unable to find {particle} mass in thermal_speed")
```

Comment:
A quick suggestion before my cat sits on my laptop.  There should be an `f` before the first quotation mark to note that it's an f-string here, ten lines below, and then around line ~500.  (`f"Unable to find {particle}..."`)

Reply:
Oh shit, yeah, the only way this is passing right now is because we're probably not testing ValueError here.

Aren't we?

Reply:
@namurphy good catch!

@StanczakDominik 
Nah, it probably just doesn't render the value and instead literally prints `{particle}`

Reply:
That it does, I just checked.